pkg/apiserver/apiserver.go

```diff
@@ -183,6 +183,7 @@ func (s *APIServer) Run(stopCh <-chan struct{}) (err error) {
 	}()
 
 	klog.V(0).Infof("Start listening on %s", s.Server.Addr)
+	klog.V(0).Infof("Open the swagger-ui from http://localhost%s/apidocs", s.Server.Addr)
```

Comment:
It would be better if we print swagger link like <http://localhost:9090/apidocs/?url=http://localhost:9090/apidocs.json/>. It will load swagger API configuration automatically.

Reply:
done